mesonbuild/compilers/compilers.py

```diff
@@ -417,6 +420,15 @@ def get_base_compile_args(options, compiler):
     # This does not need a try...except
     if option_enabled(compiler.base_options, options, 'b_bitcode'):
         args.append('-fembed-bitcode')
+    try:
+        crt_val = options['b_crtlib'].value
+        buildtype = options['buildtype'].value
+        try:
+            args += compiler.get_crt_compile_args(crt_val, buildtype)
```

Comment:
In D compilers, these should be passed as linker args, so another function like `get_crt_link_args` is required here?

Reply:
Seems like it. How about we get this merged now as is and then you can add this to your D merge request since you know more about D on Windows than any one of us?